app/modules/profilers/cpu/CPUProfiler.py

```diff
@@ -10,7 +10,7 @@
 from time import gmtime, strftime
 from datetime import datetime
 
-from modules.lepd.LepDClient import LepDClient
+from app.modules.lepd.LepDClient import LepDClient
```

Comment:
please leave it as it is,  you need to add your "app" directory to your PYTHONPATH environment variable, and you will get it work by importing from "modules...".